runtime/j9vm/j8vmi.c

```diff
@@ -129,15 +129,15 @@ JVM_CopySwapMemory(JNIEnv *env, jobject srcObj, jlong srcOffset, jobject dstObj,
 		/* The java caller has added Unsafe.arrayBaseOffset() to the offset. Remove it
 		 * here as GetPrimitiveArrayCritical returns a pointer to the first element.
 		 */
-		srcOffset -= J9VMTHREAD_CONTIGUOUS_HEADER_SIZE((J9VMThread*)env);
+		srcOffset -= J9VMTHREAD_CONTIGUOUS_INDEXABLE_HEADER_SIZE(((J9VMThread*)env));
```

Comment:
Why the extra set of parentheses around the argument? Same question for line 140.